src/Components/Movies.js

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export default class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      currentMovie: {},
+    };
+  }
+  handleChange = async (e) => {
+    const { data } = await axios.get(
+      `https://ghibliapi.herokuapp.com/films/${e.target.value}`
+    );
+    console.log(data);
+    this.setState({ currentMovie: data });
+  };
+  getMovies = async () => {
+    const { data } = await axios.get("https://ghibliapi.herokuapp.com/films");
+    console.log(data);
+    this.setState({ movies: data });
+  };
+  componentDidMount() {
+    this.getMovies();
+  }
+  render() {
+    const { movies, currentMovie } = this.state;
+    const options = movies.map((movie) => (
+      <option key={movie.id} value={movie.id}>
+        {movie.title}
+      </option>
+    ));
+
+    return (
+      <>
+        <div>
+          <h1 style={{color:"ivory"}}>Select a Movie</h1>
+          <select onChange={this.handleChange}>
```

Comment:
This should be a controlled component! (i.e. the select tag should have a value prop that allows react to control the currently selected option).